openslides_backend/services/datastore/commands.py

```diff
@@ -136,7 +136,7 @@ def data(self) -> str:
 
         class WriteRequestJSONEncoder(json.JSONEncoder):
             def default(self, o):  # type: ignore
-                if isinstance(o, FullQualifiedId):
+                if isinstance(o, str):
                     return str(o)
```

Comment:
You can just remove this check; while it was still a class, the JSON encoder did not know how to encode it, so we had to tell him manually. Now that it is a string, it should pose no problem.

Reply:
done